app/services/location_service.rb

```diff
@@ -0,0 +1,27 @@
+class LocationService
+  def initialize(zip_code)
+    @zip_code = zip_code
+  end
+  
```

Comment:
Should we add a private to this service?

Reply:
Good call! I'll add that before get_location_data.